src/components/Map/index.js

```diff
@@ -1,97 +1,66 @@
-import amaps from 'amsterdam-amaps/dist/amaps';
-
-import 'leaflet/dist/leaflet';
-import 'leaflet/dist/leaflet.css';
-import 'amsterdam-amaps/dist/nlmaps/dist/assets/css/nlmaps.css';
-
-import './style.scss';
-
-const PREVIEW_ZOOM_LEVEL = '16';
-const smallIcon = global.window.L.icon({
-  iconUrl: 'https://map.data.amsterdam.nl/dist/images/svg/marker.svg',
-  iconSize: [20, 20],
-  iconAnchor: [10, 19],
-});
-const defaultIcon = global.window.L.icon({
-  iconUrl: 'https://map.data.amsterdam.nl/dist/images/svg/marker.svg',
-  iconSize: [40, 40],
-  iconAnchor: [20, 39],
-});
-
-class Map extends React.Component {
-  componentDidMount() {
-    this.map = amaps.createMap({
-      center: {
-        latitude: this.props.latlng.latitude,
-        longitude: this.props.latlng.longitude,
-      },
-      layer: 'standaard',
-      target: 'mapdiv',
-      marker: true,
-      search: false,
-      zoom: this.props.zoom,
-    });
-
-    if (this.props.hideAttribution) {
-      this.map.attributionControl.remove();
-    }
-
-    if (this.props.hideZoomControls) {
-      this.map.zoomControl.remove();
-    }
-
-    this.renderMarker(this.props.latlng.latitude, this.props.latlng.longitude);
-  }
-
-  componentDidUpdate(prevProps) {
-    const lat = this.props.latlng.latitude;
-    const lng = this.props.latlng.longitude;
-    if (lat !== prevProps.latlng.latitude || lng !== prevProps.latlng.longitude) {
-      this.renderMarker(lat, lng);
-    }
-  }
-
-  map = {};
-
-  renderMarker(lat, lng) {
-    L.DomUtil.empty(this.map.getPane('markerPane'));
-
-    this.map.setView({ lat, lng });
-
-    const marker = L.marker(
-      [lat, lng],
-      { icon: this.props.useSmallMarker ? smallIcon : defaultIcon }
-    );
-    marker.addTo(this.map);
-  }
-
-  render() {
-    return (
-      <div className="map-component">
-        <div className="map">
-          <div className="map-container" id="mapdiv" />
-        </div>
-      </div>
-    );
-  }
-}
-
+import { ViewerContainer } from '@datapunt/asc-ui';
+import { Zoom } from '@datapunt/amsterdam-react-maps/lib/components';
+import styled from '@datapunt/asc-core';
+import { Map as MapComponent, Marker, TileLayer } from '@datapunt/react-maps';
+import { markerIcon } from 'shared/services/configuration/map-markers';
+
+const Wrapper = styled(MapComponent)`
+  height: 450px;
+  width: 100%;
+`;
+
+const StyledViewerContainer = styled(ViewerContainer)`
+  z-index: 400; // this elevation ensures that this container comes on top of the internal leaflet components
+`;
+
+const hasTouchCapabilities = !!global.L.Browser.touch;
+
+
+const Map = ({ lat, lng, mapOptions, icon, hasZoomControls, hasAttributionControl, isInteractive, ...otherProps }) => {
+  const options = {
+    ...mapOptions,
+    attributionControl: hasAttributionControl,
+    dragging: isInteractive && !hasTouchCapabilities,
+    tap: isInteractive && !hasTouchCapabilities,
+    scrollWheelZoom: isInteractive && !hasTouchCapabilities,
```

Comment:
What is the effect of these settings on both desktop and mobile? Is it still possible to drag the map with the mouse and to do a two-finger scroll and zoom?
In a previous POC, both `tap` and `scrollWheelZoom` where set to 'false' to achieve that effect.